crates/store/re_types/src/components/lat_lon_ext.rs

```diff
@@ -0,0 +1,47 @@
+use crate::datatypes::DVec2D;
+
+use super::LatLon;
+
+// ---
+
+impl LatLon {
+    /// The origin.
+    pub const ZERO: Self = Self::new(0.0, 0.0);
```

Comment:
That doesn't really make sense for lat/lon. That's just a point in the ocean at the equator